grodecoder.py

```diff
@@ -1089,10 +1089,18 @@ def is_lipid(
         ]
         if "formula_no_h" in dict_count.keys():
             formula_graph = dict_count["formula_no_h"]
+
+
+            # Sometimes the file alias and database alias do not match (because the GRO file trims 
```

Comment:
Proposition:

```
# Sometimes, the residue name found in the database does not match 
# with the one found in the strcture file, because:
# - in GRO files, residue names are trimed at 4 characters, 
# - in PDB files, residue names are trimed at 3 characters.
# Thus we check that rsidue name (caled alias) from the database starts 
# with the residue name from the structure file.
# In case of ambiguity (for instance SB3-10, SB3-12, and SB3-14,
# all trimed at SB3-1 in a GRO file, the formula is used to solve this ambiguity.
```

Reply:
@KarinDuong are you sure `SB3-1` is a proper residue name in a GRO file? Because it's 5 characters long.

Reply:
For gro file, the resname is 5 characters (https://manual.gromacs.org/archive/5.0.3/online/gro.html)

Reply:
Ok, could you:
- Update the text of the comment?
- Add the above Gromacs link in the comment
- Verify in the PDB doc whether residue name has 3 or 4 characters in a PDB file?

Reply:
For PDB, it's 3 
`Residue names are a maximum of three characters long[§](https://www.cgl.ucsf.edu/chimera/docs/UsersGuide/tutorials/pdbintro.html#note5) and uniquely identify the residue type. Thus, all residues of a given name should be the same type of residue and have the same structure (contain the same atoms with the same connectivity).` 
(https://www.cgl.ucsf.edu/chimera/docs/UsersGuide/tutorials/pdbintro.html )

Reply:
Done